pkg/crc/network/nameservers.go

```diff
@@ -34,13 +36,48 @@ func GetResolvValuesFromInstance(sshRunner *ssh.Runner) (*ResolvFileValues, erro
 func CreateResolvFileOnInstance(sshRunner *ssh.Runner, resolvFileValues ResolvFileValues) error {
 	resolvFile, _ := CreateResolvFile(resolvFileValues)
 
+	sd := systemd.NewInstanceSystemdCommander(sshRunner)
+	// Check if ovs-configuration.service exist and if not then it is old bundle and use the same way to
+	// update resolve.conf file
+	if state, err := sd.Status("ovs-configuration.service"); err != nil || state == states.NotFound {
+		if err := replaceResolvConfFile(sshRunner, resolvFile); err != nil {
+			return fmt.Errorf("error updating resolve.conf file: %s", err)
+		}
+		return nil
+	}
+
+	if err := sd.Start("ovs-configuration.service"); err != nil {
+		return err
+	}
+
+	nameservers := strings.Join(resolvFileValues.GetNameServer(), ",")
+	searchDomains := strings.Join(resolvFileValues.GetSearchDomains(), ",")
```

Comment:
I'd do this in `updateNetworkManagerConfig`

Reply:
Done